icu4j/main/core/src/main/java/com/ibm/icu/util/ChineseCalendar.java

```diff
@@ -1155,6 +1171,12 @@ protected int internalGetMonth(int defaultValue)
         return internalGetMonth();
     }
 
+    /**
+     * {@inheritDoc}
+     *
+     * @stable ICU 2.8
+     */
+    @Override
```

Comment:
This is the "real" change here. This override method was added in
- https://github.com/unicode-org/icu/pull/3348

The parent method Calendar.getActualMaximum() is stable ICU 2.0, but the whole class ChineseCalendar is from 2.8, so I think that makes sense as a version number here.